src/eventer/eventer_epoll_impl.c

```diff
@@ -456,9 +456,10 @@ static int eventer_epoll_impl_loop(int id) {
   (void)id;
   struct epoll_event *epev;
   struct epoll_spec *spec;
+  int max_fds_at_once = 1024;
```

Comment:
maxfds is defined based on the particulars of the OS (if i'm reading eventer_impl_setrlimit correctly).  While probably unlikely anywhere our stuff is running, is it possible for an rlimit to be less than 1024?  And if so, would that break when you pass epoll_wait a value that large?

Reply:
No risk. This just sets the limit of the size of output events produced by epoll_wait.